modules/blocks.php

```diff
@@ -190,3 +191,60 @@ function jetpack_slideshow_block_load_assets( $attr, $content ) {
 	Jetpack_Gutenberg::load_assets_as_required( 'slideshow', $dependencies );
 	return $content;
 }
+
+jetpack_register_block(
```

Comment:
Could you add a comment here introducing the docblock, like it's done for the other blocks?